chain-tx-enclave-next/mls/src/main.rs

```diff
@@ -9,15 +10,25 @@ fn main() -> io::Result<()> {
     use rs_libc::alloc::*;
     use rustls::internal::msgs::codec::Codec;
 
-    let kp = OwnedKeyPackage::new(
-        EnclaveRaContext::new(&EnclaveRaConfig {
-            sp_addr: "0.0.0.0:8989".to_owned(),
-            certificate_validity_secs: 86400,
-        })
-        .unwrap(),
-    )
-    .unwrap();
-    io::stdout().write_all(&kp.keypackage.get_encoding())
+    let config = EnclaveRaContext::new(&EnclaveRaConfig {
+        sp_addr: "0.0.0.0:8989".to_owned(),
+        certificate_validity_secs: 86400,
+    });
+    if config.is_err() {
+        println!("cannot connect ra-sp-server, run ra-sp-server beforehand e.g.) ra-sp-server --quote-type Unlinkable --ias-key $IAS_API_KEY --spid $SPID")
+    }
+    let kp = OwnedKeyPackage::new(config.unwrap()).unwrap();
+
+    let now = chrono::Utc::now().timestamp() as u64;
+    let verication_result = kp
+        .keypackage
+        .verify(&*ra_client::ENCLAVE_CERT_VERIFIER, now);
+    if let Err(value) = verication_result {
+        println!("verification_fail {}", value);
```

Comment:
Write the error into the stderr, and exit with a non zero code.

Reply:
ok